README.rst

```diff
@@ -123,17 +123,16 @@ server with hot reload using::
 Switching between the webpack dev server and serving from ``static/`` requires a
 restart of your Django dev server to pick up the changed file paths.
 
-You must also configure Solr and install the configSet found under ``solr_conf``.
-If a core does not exist, and the configSet is installed correctly, an appropriate
-core will be made for you. To install the configSet::
+Copy the project Solr configset in ``solr_conf`` into your solr server configset
+directory. For a local install::
 
-    (using root privileges for chown and permission as needed)
-    cp -r solr_conf /path/to/solr/server/solr/configsets/sandco
-    chown solr:solr -R /path/to/solr/server/solr/configsets/sandco
+    cp -r solr_conf /path/to/solr/server/solr/configsets/shxco
+    chown solr:solr -R /path/to/solr/server/solr/configsets/shxco
 
-Note that this location will vary if your Solr instance has a separate data
-folder. See ``DEPLOYNOTES`` for an example of that setup, which is commmon on
-server installs of Solr.
+Create Solr collection with the configured configset (use create_core with
+Solr standalone and create_collection with SolrCloud)::
```

Comment:
If this is the only difference, we don't need to expand, but should we have centralized SolrCloud documentation somewhere?

Reply:
I can't imagine we're ever going to run solr cloud locally for dev work... Basically, I want to leave all the solr cloud management to PUL and automate our side of it for the deploys.

I don't know how much this matters, but sometimes it seems like the core/collection terminology is confusing and it seemed like a good thing to note. What do you think, remove?

Reply:
Ooo yeah, I can't imagine ever wanting to run solr cloud locally 😄 I do think that this piece of information is important, but I guess it will be documented in our ansible scripts? Should we make a ticket on the CDH dev site to collect our SolrCloud notes there?